packages/ra-ui-materialui/src/layout/TitleForRecord.tsx

```diff
@@ -0,0 +1,18 @@
+import React, { FC } from 'react';
+import PropTypes from 'prop-types';
+import Title, { TitlePropType, TitleProps } from './Title';
+
+interface Props extends TitleProps {
+    record: TitleProps['record'];
+}
+
+const TitleForRecord: FC<Props> = props =>
+    props.record ? <Title {...props} /> : null;
```

Comment:
the previous version white listed the props it passed to the `<Title>` component. Please keep that behavior.